### DISCUSSION TITLE
Eyedrop addition

### DISCUSSION THREAD

**Discussion by Beanable** [ScratchAddons/ScratchAddons#5070] [Ideas] [answered]
There should be a feature that you can use the eyedrop in the costume editor page and be able to get the color from a color on the scratch canvas.

**Comment by GarboMuffin** [comment, marked as answer]
https://github.com/ScratchAddons/ScratchAddons/issues/4998

https://github.com/ScratchAddons/ScratchAddons/pull/5002
